src/Components/PollList/PollList.jsx

```diff
@@ -114,27 +149,76 @@ const PollList = () => {
               <TableCell>{row.poll_question}</TableCell>
               <TableCell>{row.start_date}</TableCell>
               <TableCell>{row.end_date}</TableCell>
-              <TableCell>{row.participants}</TableCell>
-              <TableCell
-                className={classes.status}
-                style={{
-                  backgroundColor:
-                    (row.status === "LIVE" && "#ffc2c2") ||
-                    (row.status === "CREATE" && "#99ff99") ||
-                    (row.status === "ENDED" && "#c1c0c0"),
-                  color:
-                    (row.status === "LIVE" && "#ff4141") ||
-                    (row.status === "CREATE" && "#006622") ||
-                    (row.status === "ENDED" && "#000000"),
-                }}
-              >
-                {row.status}
+              <TableCell align="center">{row.participants}</TableCell>
+              <TableCell align="center">
+                <Typography
+                  className={classes.status}
+                  style={{
+                    backgroundColor:
+                      (row.status === "LIVE" && "#ffc2c2") ||
+                      (row.status === "CREATE" && "#99ff99") ||
+                      (row.status === "ENDED" && "#c1c0c0"),
+                    color:
+                      (row.status === "LIVE" && "#ff4141") ||
+                      (row.status === "CREATE" && "#006622") ||
+                      (row.status === "ENDED" && "#000000"),
+                  }}
+                >
+                  {row.status}
+                </Typography>
               </TableCell>
               <TableCell>
                 {row.action}
-                <Button variant="outlined" startIcon={<DeleteIcon />}>
+                <Button
+                  variant="outlined"
+                  startIcon={<DeleteIcon />}
+                  onClick={handleClickOpen}
+                >
                   Delete
                 </Button>
+                <Dialog
+                  open={open}
+                  onClose={handleClose}
+                  aria-labelledby="alert-dialog-title"
+                  aria-describedby="alert-dialog-description"
+                >
+                  <DialogTitle id="alert-dialog-title" align="center">
+                    <Box
+                      component="img"
+                      sx={{
+                        height: 400,
+                        width: 250,
+                        maxHeight: { xs: 400, md: 170 },
+                        maxWidth: { xs: 250, md: 200 },
+                      }}
+                      alt=""
```

Comment:
nếu là thẻ <img> thì e nên thêm attribute alt nhé, cái này nó ko hiện lỗi lên nhưng khi ảnh ko load đc nó sẽ dùng text này thay thế + hỗ trợ ng mù khi dùng app (cái này app xịn mới có nhưng mình cứ theo best practice)
```suggestion
                      alt="delete-poll-image"
```